src/ui/components/MobileHeaderPreview/MobileHeaderPreview.scss

```diff
@@ -0,0 +1,31 @@
+.mobile-preview-header {
+  display: none;
+  width: 100%;
+  background-color: transparent;
+  justify-content: space-between;
+  align-items: center;
+  padding: 13px 32px;
```

Comment:
@Sotatek-DukeVu What I was asking here is that you will add a comment for the use of `px` vs `rem` and not for the `z-index`.

Reply:
Ah I'm sorry. I forgot using rem in this case. Because I followed Figma design and you know all of them using `px` as unit in dev mode. Sorry for my mistake.